package.json

```diff
@@ -10,7 +10,15 @@
   "author": "Clinton Enyinna",
   "license": "ISC",
   "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11"
```

Comment:
I guess you slightly miss adding `webpack-dev-server` in here;
![image](https://user-images.githubusercontent.com/31897434/81985810-a6d72100-962e-11ea-83ec-933430b828f8.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/